### DISCUSSION TITLE
Kotlin for app & plugin & capacitor codebase?

### DISCUSSION THREAD

**Discussion by IronGeek** [ionic-team/capacitor#6109] [Q&A]
Hi,

Related to the issue #1227 asked more than 3 years ago,
Does the ionic-team (still) have any plan to migrate the capacitor (android) codebase to Kotlin?

**Comment by ximex** [comment]
migration documentation is here: https://developer.android.com/build/migrate-to-kotlin-dsl#syntax-conversion
and is it also planed to migrate away from [buildscript](https://developer.android.com/build/migrate-to-kotlin-dsl#migrate-buildscript)?
